ctapipe/reco/ImPACT.py

```diff
@@ -161,12 +169,17 @@ def initialise_templates(self, tel_type):
 
             self.prediction[tel_type[t]] = \
                 TemplateNetworkInterpolator(self.root_dir + "/" +
-                                            self.file_names[tel_type[t]][0])
+                                            self.file_names[tel_type[t]][0],
```

Comment:
use pathlib here for filename generation.  If root_dir is a `Path`, you can just do: `rootdir / self.file_names[tel_type[t]][0]`.  That way it will not break on systems like Windows (on which surprisingly ctapipe works fine so far!)

Reply:
How do you know? Related to https://github.com/cta-observatory/pyeventio/issues/109#issuecomment-492659009? :rofl:

Reply:
My office mate is running it on windows! I was pretty surprised that it all worked (even pyeventio), if installed via pip.

Reply:
You also need to register any files you open with the Provenance system (if not using `get_dataset_path`, which does it automatically).  To do that, just do:
```py3
from ctapipe.core import Provenance
PROV = Provenance()
...
PROV.add_input_file(filename, role="ImPACT Template file")
```